upgrade_logic/business_objects/gateway.py

```diff
@@ -1,5 +1,21 @@
 import re
 from submodules.model.business_objects import attribute, embedding, user, general
+from submodules.model import enums
+
+def gateway_1_6_1() -> bool:
+    __gateway_1_6_1_add_attribute_visibility()
+    return True
+
+
+def __gateway_1_6_1_add_attribute_visibility() -> bool:
+    sql = f"""
+    UPDATE attribute
+    SET visibility = '{enums.AttributeVisibility.DO_NOT_HIDE.value}'
+    WHERE visibility IS NULL
+    """
+    general.execute_sql(sql)
+    general.commit()
+    return True
```

Comment:
we usually include an information in the log if something was changed 
if you don't want to do this with the orm model counting first for IS NULL should do the trick

Reply:
Added count, check and print statement